controllers/articles.js

```diff
@@ -1,9 +1,24 @@
 
 var mongoose  = require('mongoose'),
-  moment    = require('moment'),
-  Article   = mongoose.model('Article');
+  moment      = require('moment'),
+  Article     = mongoose.model('Article'),
+  passport    = require('passport'),
+  express     = require('express'),
+  auth        = require('./../middlewares/auth');
 
-exports.view = function (req, res) {
+module.exports = (function () {
```

Comment:
This reasoning just doesn’t make sense:

  1. `module` (and `exports`) are local to each file, not globals ([source][1])
  2. the functions you declare afterwards are top-level

  [1]: http://nodejs.org/docs/latest/api/modules.html#modules_the_module_object